pkg/configuration/get_configuration.go

```diff
@@ -0,0 +1,45 @@
+package configuration
```

Comment:
I guess it could be just `pkg/configuration/get.go`

Reply:
Done in 0c83f63f5b77f7e1b93559b5cffc8ff2cdc21725

I also renamed load_configuration.go to load.go for consistency.